deployment-worker/repositories/deployment.py

```
@@ -0,0 +1,73 @@
+from database.connection import getConnection
+from models.deployment import Deployment, DeploymentStatus
```

Ruff (0.12.2)

`models.deployment.Deployment` imported but unused

Remove unused import

(F401)